src/plugins/warnings/index.tsx

```diff
@@ -3,7 +3,7 @@ import { AlertActionButton, AlertModal } from '@revenge-mod/modules/common/compo
 import { ClientInfoModule } from '@revenge-mod/modules/native'
 import { registerPlugin } from '@revenge-mod/plugins/internals'
 
-const MinimumSupportedBuildNumber = ReactNative.Platform.select({ android: 260000, ios: 69233 })!
+const MinimumSupportedBuildNumber = ReactNative.Platform.select({ android: 263000, ios: 69420 })!
```

Comment:
why was this changed?

Reply:
This was never meant to be pushed, or at least was meant to be pushed with the combination of the "support 263.x" change. However, iOS 263.x hasn't been released yet.